unlock-protocol.com/blog/version-1-2.md

```diff
@@ -0,0 +1,87 @@
+
+![Etherscan](/static/images/blog/version-1-2/etherscan.png)
+
+## The Proxy
+
+First, you can see that you can actually interact with both the proxy, but also the implementation directly. The proxy part offers the ability to `upgradeTo` a new version (by proving the address of the new implementation), `upgradeToAndCall` which does the same but also invoke a custom function right after the upgrade, or `changeAdmin` which would change the admin. All of these functions can only be called by the admin: don't waste gas calling them yourself!
+
+As of last Monday, the [proxy's admin](https://etherscan.io/address/0x79918A4389A437906538E0bbf39918BfA4F7690e) [upgraded the implementation](https://etherscan.io/tx/0x358335670ef14c1deb72b3711eb0dd629ebb3801f1aa2b64a4feb04df00dc139) to use a new one, [deployed at this address](https://etherscan.io/address/0xe36793f0b4db71ff0a5216412f80ba89b2927445).
+
+## Unlock
+
+More importantly, you can interact with the Unlock smart contract itself. For example, you can view view [the owner of the Unlock contract](https://etherscan.io/address/0xa39b44c4affbb56b76a1bf1d19eb93a5dfc2eba9), the global token symbol for all NFT: `KEY` or even the current version of the Unlock contract (we use `5` internally and `1.2` externally).
+
+Another very important element that you can view is the _Gross Network Product_ which is the total sum of ethereum spent to purchase lock keys, as long as the key was priced in Eth, DAI, SAI or BAT. We use Uniswap to get a price feed when we need to convert to Ether, but that's for another post ;) Measuring the Gross Network Product is a critical piece of the [Unlock Discount Token](https://github.com/unlock-protocol/unlock/wiki/The-Unlock-Discount-Tokens) model that we are working on to incentivize membership referrals.
+
+By using Etherscan you can also deploy locks, without having to go through [our Dashboard](https://app.unlock-protocol.com/dashboard/), using the `createLock` method directly. The new version deployed on Monday uses the new Ethereum `create2` and leverages [eip-1014](https://eips.ethereum.org/EIPS/eip-1014) under the hood. This allows creators to deploy locks at only 15% of the cost that it took before that! In order to achieve this, we had to deploy a so [called `template` contract](https://etherscan.io/address/0x2d5487420fbeb5ba74eadf51084d4f71e1733983) which provides the reference implementation for locks to leverage.
+
+
+# The new public locks
+
+I want to repeat that old the **locks deployed before Monday have stayed absolutely unchanged**. It is actually impossible to change them. Once deployed, their code cannot be altered. This is a very important part of the "decentralization promise" that we make to creators: nobody will be able to change the terms of your memberships, but them.
+
+Now, if someone deploys a lock now, it comes with new features and improvements. Here are a few of the most exciting ones.
```

Comment:
Maybe remove one instance of "now"
>Now, if someone deploys a lock now, 

"If someone deploys a lock now, it comes with..."